aten/src/ATen/native/native_functions.yaml

```diff
@@ -2915,7 +2915,6 @@
   variants: function, method
   dispatch:
     QuantizedCPU: quantized_index
-  tags: dynamic_output_shape
```

Comment:
index also implements masked index, so the tag is correct.

Reply:
Hrm that's annoying. dynamo graph breaks on dynamic output shapes, so we can't use it in decompositions otherwise. I wonder how `Tensor.__getitem__` works then.